cincinnati/src/plugins/internal/graph_builder/github_openshift_secondary_metadata_scraper/plugin.rs

```diff
@@ -445,6 +448,10 @@ impl PluginSettings for GithubOpenshiftSecondaryMetadataScraperSettings {
 #[async_trait]
 impl InternalPlugin for GithubOpenshiftSecondaryMetadataScraperPlugin {
     async fn run_internal(self: &Self, mut io: InternalIO) -> Fallible<InternalIO> {
+        get_tracer()
+            .get_active_span()
+            .update_name("github-secondary-metadata".to_string());
```

Comment:
Use the predefined constant instead:
```suggestion
            .update_name(Self::PLUGIN_NAME.to_string());
```